src/signal-safe.cpp

```diff
@@ -7,6 +7,11 @@
 #include <unistd.h>
 #include <cstring>
 
+// clang-format off
```

Comment:
I have no clue why clang-format mangles unrelated lines here but not in https://github.com/hyprwm/Hyprland/blob/80a4852266643c622ec68969b39c5abc3c610f30/src/config/ConfigManager.cpp#L36-L41

Reply:
Wouldn't that disable `NOLINTNEXTLINE`? I can't test clang-tidy atm.

Reply:
if you put it _below_ extern C, no